### PR TITLE
Fixes issue with listing facts by resource group in azure_rm_load_balancer_facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Fixing issue with listing facts by resource group.
Changed to new flattened format of returning facts.
Added tesst

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
azure_rm_loadbalancer_facts

##### ANSIBLE VERSION
2.4

### CODE REVIEW

**Comment by yuwzho** [comment]
shipit

File: test/integration/targets/azure_rm_loadbalancer_facts/aliases [hunk 1]
```diff
@@ -0,0 +1,2 @@
+cloud/azure
```
Comment (mattclay) [inline test/integration/targets/azure_rm_loadbalancer_facts/aliases:1]: Add `posix/ci/cloud/group5/azure` to run the tests in CI.

**Comment by Akasurde** [comment]
@zikalino Can you please include changes suggested by @mattclay so that we can take this forward?

needs_info

**Comment by yuwzho** [comment]
@mattclay Could you please show any guide line of the aliases? When we add the integration test, we have no idea what should be added but to copy from other integration test.

**Comment by mattclay** [comment]
@yuwzho I've started working on some tests to help enforce correct aliases as well as accompanying documentation.

**Comment by mattclay** [comment]
@yuwzho The improved sanity tests for aliases have been merged. The documentation is here:

https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/integration-aliases.html

Please let me know if that documentation helps, if you have questions, suggestions, etc.

**Comment by yuwzho** [comment]
@mattclay Thanks a lot

**Pull request closed without merging**